authorizer-app/src/app/services/rest-source-user.service.ts

```diff
@@ -10,27 +10,29 @@ import {environment} from "../../environments/environment";
 export class RestSourceUserService {
 
   private serviceUrl = environment.BACKEND_BASE_URL + "/users";
-  constructor(private http: HttpClient) { }
+
+  constructor(private http: HttpClient) {
+  }
 
   getAllUsers(): Observable<RestSourceUser[]> {
     return this.http.get<RestSourceUser[]>(this.serviceUrl);
   }
 
   updateUser(sourceUser: RestSourceUser): Observable<any> {
-
-    return this.http.post(this.serviceUrl + '/' + sourceUser.id, sourceUser);
+    const params = new HttpParams().set("validate", String(environment.VALIDATE));
```

Comment:
why not validate for all environments?

Reply:
I think this will get the value for `VALIDATE` based on the current environment.

Reply:
Yes. But i am curious why do we optionally validate instead of always?

Reply:
For example, we might not want to validate in a development environment. Just to make it more configurable